src/Twilio/Clients/TwilioRestClient.cs

```diff
@@ -53,13 +58,16 @@ public class TwilioRestClient : ITwilioRestClient
         /// <param name="region">region to make requests for</param>
         /// <param name="httpClient">http client used to make the requests</param>
         /// <param name="edge">edge to make requests for</param>
+        /// <param name="logLevel">log level for http logging</param>
+
         public TwilioRestClient(
             string username,
             string password,
             string accountSid = null,
             string region = null,
             HttpClient httpClient = null,
-            string edge = null
+            string edge = null,
+            string logLevel = null
```

Comment:
Should limit the number of constructor parameters. I think it's fine just to use a setter.